src/libs/kata-types/src/volume.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Volume to support dm-verity over block devices.
+pub const KATA_VOLUME_TYPE_DMVERITY: &str = "dmverity";
+
+/// Key to indentify directory creation in `Storage.driver_options`.
+pub const KATA_VOLUME_OVERLAYFS_CREATE_DIR: &str =
+    "io.katacontainers.volume.overlayfs.create_directory";
```

Comment:
Please help to add a blank line here to seperate overlayfs from dm-verity.

Reply:
Or move  line 10-11 below line 25 with a blank line.

Reply:
Fixed.